### PR TITLE
(CDAP-5047) Added a Batch Source for Azure Blob Store based on FileBa…

### PR DESCRIPTION
**Pull request by bdmogal**
…tchSource

Jira: [CDAP-5047](https://issues.cask.co/browse/CDAP-5047)

### CODE REVIEW

File: core-plugins/docs/AzureBlobStore-batchsource.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Example
+-------
+
+This example connects to Azure Blob Store and reads in files found in the specified directory. This example uses
+the 'hydrator' container on the 'hydratorstorage.blob.core.windows.net' account on Azure Blob Store
+
+    {
+        "name": "AzureBlobStore",
+        "properties": {
+            "path": "/path/to/input",
```
Comment (sreevatsanraman) [inline core-plugins/docs/AzureBlobStore-batchsource.md:36]: would this have wasb:// or no?

**Comment by sreevatsanraman** [comment]
one minor comment. LGTM otherwise

File: core-plugins/docs/AzureBlobStore-batchsource.md (hunk 1 above)
Comment (bdmogal) [inline core-plugins/docs/AzureBlobStore-batchsource.md:36]: nope. the plugin constructs it [here](https://github.com/caskdata/hydrator-plugins/pull/163/files#diff-bb0d85323b4f4478fb6734970284444bR60)

**Comment by sreevatsanraman** [comment]
Lgtm

**Pull request merged**